randobot/zsr.py

```diff
@@ -64,15 +64,18 @@ class ZSR:
     def __init__(self, ootr_api_key):
         self.ootr_api_key = ootr_api_key
         self.presets = self.load_presets()
-        self.presets_dev = self.load_presets_dev()
+        self.presets_dev = self.load_presets(dev=True)
         self.last_known_dev_version = None
         self.get_latest_dev_version()
 
-    def load_presets(self):
+    def load_presets(self, dev=False):
         """
         Load and return available seed presets.
         """
         settings = requests.get(self.settings_endpoint).json()
+        if dev:
+            settings = requests.get(self.settings_dev_endpoint).json()
```

Comment:
This effectively forces the load twice even for dev - i would change this to if else.